src/entities/virtual-currency.ts

```diff
@@ -35,6 +35,6 @@ export function toVirtualCurrency(
     balance: virtualCurrencyResponse.balance,
     name: virtualCurrencyResponse.name,
     code: virtualCurrencyResponse.code,
-    serverDescription: virtualCurrencyResponse.description,
+    serverDescription: virtualCurrencyResponse.description ?? null,
```

Comment:
Fixes a bug where if the `description` field was omitted in the response (our backend does this), `serverDescription` would be set to `undefined` instead of `null`